crates/broker/src/order_monitor.rs

```diff
@@ -518,12 +518,14 @@ where
                             match err {
                                 OrderMonitorErr::UnexpectedError(inner) => {
                                     tracing::error!(
-                                        "Failed to lock order: {order_id} - {inner:?}",
+                                        "Failed to lock order: {order_id} - {} - {inner:?}",
+                                        err.code()
                                     );
                                 }
                                 _ => {
                                     tracing::warn!(
-                                        "Soft failed to lock request: {order_id} - {err:?}",
+                                        "Soft failed to lock request: {order_id} - {} - {err:?}",
+                                        err.code()
```

Comment:
This one _seems_ unnecessary because the code should be in `err` debug though, no?